database/factories/AddressFactory.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @var \Illuminate\Database\Eloquent\Factory $factory
+ */
+
+use Faker\Generator as Faker;
+use Tychovbh\Mvc\Address;
+
+/*
+|--------------------------------------------------------------------------
+| Model Factories
+|--------------------------------------------------------------------------
+|
+| This directory should contain each of the model factory definitions for
+| your application. Factories provide a convenient way to generate new
+| model instances for testing / seeding your application's database.
+|
+*/
+
+$factory->define(Address::class, function (Faker $faker) {
+    return [
+        'zipcode' => '2352 CZ',
```

Comment:
use faker voor generating zipcode, housenumber and country. Now all  database records will be generated with the same values. If you need a real zipcode, housenumber for pdok test than overwrite the values in testcase:
```php 
$address = factory(Address::class)->create([
    'zipcode' => '2351CZ',
    ....
])
```